content/docs/concepts/replication/_index.md

```diff
@@ -24,6 +24,7 @@ Container Storage Modules for Replication provides the following capabilities:
 | <div style="text-align: left">Asynchronous block volume replication                                                                                               |   N/A    |    N/A     |    Yes    |    Yes    |  No   |
 | <div style="text-align: left">Synchronous file volume replication                                                                                                 |   No     |     No     |    No     |    No     |  No   |
 | <div style="text-align: left">Synchronous block volume replication                                                                                                |   N/A    |     N/A    |    No     |    Yes    |  No   |
+| <div style="text-align: left">Shared NFS volume replication                                                                                                       |   N/A    |    Yes     |    N/A    |    N/A    |  N/A  |
```

Comment:
Should this not be PowerStore vs PowerScale? Also why is this "Shared NFS volume replication" vs "Shared NFS Volumes"?

Reply:
Fixed to set as PowerStore and as Shared NFS Volumes. After rebasing with main, the order of arrays in the table was changed.